### PR TITLE
Restructured cloud org and user management

### PR DESCRIPTION
**Pull request by sanderson**
Closes influxdata/DAR#257

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud/organizations/users.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+- [Users management page](#users-management-page)
+- [Invite a user to your organization](#invite-a-user-to-your-organization)
+  - [Resend an invitation](#resend-an-invitation)
+  - [Withdraw an invitation](#withdraw-an-invitation)
+- [Remove a user from your organization](#remove-a-user-from-your-organization)
+  - [Remove yourself from an organization](#remove-yourself-from-an-organization)
+
+## Users management page
+Manage your organization's users from your organizations **Users management page**.
```
Comment (jstirnaman) [inline content/influxdb/cloud/organizations/users.md:30]:
```suggestion
Manage your organization's users from your organization's **Users management page**.
```

File: content/influxdb/cloud/organizations/users.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+{{< nav-icon "account" >}}
+
+## Invite a user to your organization
+
+1.  Navigate to your organization's [Users management page](#users-management-page).
+2.  Under **Add a new user to your organization**, enter the email address of
+    the user to invite and select their role in your organization.
+
+    {{% note %}}
+#### Available roles
```
Comment (jstirnaman) [inline content/influxdb/cloud/organizations/users.md:43]: Is the indentation in lines 42-47 intentional?

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Minor fixes, otherwise looks good.

File: content/influxdb/cloud/organizations/users.md (hunk 2 above)
Comment (sanderson) [reply content/influxdb/cloud/organizations/users.md:43]: Yeah, this nests the note block under the list item. Content inside of shortcodes doesn't know about it's surrounding context or where the shortcode falls within the indentation structure. So whenever you want to nest a shortcode inside of a line item, the shortcode itself should be nested under the line item, but the content inside the shortcode needs to be formatted without indentation. If it's indented, Hugo parses it as a code block (default markdown behavior).

**Review by jstirnaman — APPROVED** [review]

**Pull request merged**